doc/develop/test/ztest.rst

```diff
@@ -114,27 +114,27 @@ Test result expectations
 ========================
 
 Some tests were made to be broken. In cases where the test is expected to fail or skip due to the
-nature of the code, it's possible to annotate the test as such. For example::
+nature of the code, it's possible to annotate the test as such. For example:
 
   .. code-block:: C
```

Comment:
```suggestion
.. code-block:: C
```

Reply:
Why, I use the same indentation as the code-block below this change, we need to make code consistent.

Reply:
it's ok, but file should be fixed at some point, such extra indentation is not needed.